### ISSUE TITLE
Alt-S shortcut doesn't enable Surfingkeys

### ISSUE THREAD

**Issue by aspiers** [brookhong/Surfingkeys#382]
If I hit Alt-S then the popup says "Surfingkeys turned OFF for ..." every time.  It no longer toggles between off and on.

This is at least the 6th time that the enable/disable feature has broken in some way (see #167, #168, #169, #170, and #182), and that makes me worry about the stability of Surfingkeys in general.  Is it possible to implement some regression tests to prevent it from breaking again in the future?

**Comment by brookhong** [comment]
Thanks, will think about regression tests.

But for this issue, I could not reproduce it, could you provide more info for debugging? Such as console error from inspect tool.

**Comment by aspiers** [comment]
There are no errors on the console.  What else can I check?

**Comment by brookhong** [comment]
With latest version -- 0.9.23, do you still have this problem?

**Comment by aspiers** [comment]
Yes I still have the problem, but I think I just figured out why.  Am I right in thinking that `Alt-S` only toggles the *per-site* enable/disable, whereas the enable/disable on the dropdown menu from the icon is global?  If I'm right, I can explain the problem: I think I had it globally disabled, so it didn't make any difference whether I toggled the per-site setting via `Alt-S`.

This could be made a lot clearer in the UI by changing the message shown when you press `Alt-S` so that if it's globally disabled, SurfingKeys warns about that and explains how to globally re-enable. The UI should always follow https://en.wikipedia.org/wiki/Principle_of_least_astonishment ;-)
